src/components/badge/bl-badge.css

```diff
@@ -0,0 +1,52 @@
+:host {
+  display: inline-block;
+  max-width: 100%;
+
+  --bl-badge-bg-color: var(--bl-color-accent-primary-background);
+  --bl-badge-color: var(--bl-color-primary);
+  --bl-badge-font: var(--bl-size-2xs);
```

Comment:
I think this value is wrong.

Reply:
you are right this value is wrong i will fix the "--bl--badge-font" value.